grammar.js

```diff
@@ -1109,6 +1123,12 @@ module.exports = grammar({
       $.keyword_leakproof,
     ),
 
+    function_security: $ => seq(
```

Comment:
whats the purpose of this? Is this related to the string parsing?

Reply:
Not related but I come across it while making tests.

Reply:
Please create a separate commit for these kind of things in the future. Otherwise git history becomes meaningless.